packages/react-components/react-nav-preview/src/components/NavCategoryItem/renderNavCategoryItem.tsx

```diff
@@ -18,13 +18,9 @@ export const renderNavCategoryItem_unstable = (
   return (
     <NavCategoryItemProvider value={contextValues.navCategoryItem}>
       <state.root>
-        {/* TODO: These were copied from AccordionHeader.
-        We need to decide if they are still applicable
-        in our scenario and how to adapt them. */}
-        {/* <state.button> */}
-        {/* Todo: {state.icon && <state.icon />} */}
+        {state.icon && <state.icon />}
         {state.root.children}
```

Comment:
This one should be the content, right? we forward the children to the content.

```suggestion
        <state.content/>
```

One thing to keep in mind is that maybe we should keep the previous implementation unless we have a specific reason for adding a DOM element.

Reply:
good catch - must be some left over crust from drafting off previous implementations. Went through and removed all the 'content' divs throughout.